src/arduino-nano-element.ts

```diff
@@ -0,0 +1,277 @@
+          </g>
+          <rect x="${15 * 2.54}" y="${2 * 2.54}" width="${2 * 2.54}" height="${3 * 2.54}" />
+        </g>
+
+        <!-- USB mini type B -->
+        <g id="usb-mini-b" stroke-width=".1" paint-order="stroke fill">
+          <g fill="#333" filter="url(#solderPlate)">
+            <rect x=".3" y="3.8" width="1.6" height="9.8" />
+            <rect x="5.5" y="3.8" width="1.6" height="9.8" />
+            <rect x="7.3" y="7.07" width="1.1" height=".48" />
+            <rect x="7.3" y="7.82" width="1.1" height=".48" />
+            <rect x="7.3" y="8.58" width="1.1" height=".48" />
+            <rect x="7.3" y="9.36" width="1.1" height=".48" />
+            <rect x="7.3" y="10.16" width="1.1" height=".48" />
+          </g>
+          <rect x="-1.4" y="4.8" width="8.9" height="7.8" fill="#999" stroke-width=".26" />
+          <rect x="-1.25" y="5" width="8.6" height="7.4" fill="#ccc" stroke="#bbb" />
+          <g fill="none" stroke="#333" stroke-width=".26" stroke-linecap="round">
+            <path d="m2.6 5.9h-3.3v0.9h3.3m0 3.7h-3.3v0.9h3.3M-0.6 7.6l4.3 0.3v1.5l-4.3 0.3" />
+            <path d="m3.3 7.9v1.5" stroke-width="1" stroke-linecap="butt" />
+            <path d="m6 6.4v4.5" stroke-width=".35" />
+          </g>
+        </g>
+
+        <!-- LEDs -->
+        <g transform="translate(27.7 5)">
+          <use xlink:href="#led-body" />
+          ${ledTX &&
+          svg`<circle cx="1.3" cy=".55" r="1.3" fill="#ff8080" filter="url(#ledFilter)" />`}
+        </g>
+        <g transform="translate(27.7 7)">
+          <use xlink:href="#led-body" />
+          ${ledRX &&
+          svg`<circle cx="1.3" cy=".55" r="1.3" fill="#80ff80" filter="url(#ledFilter)" />`}
+        </g>
+        <g transform="translate(27.7 9)">
+          <use xlink:href="#led-body" />
+          ${ledPower &&
+          svg`<circle cx="1.3" cy=".55" r="1.3" fill="#80ff80" filter="url(#ledFilter)" />`}
+        </g>
+        <g transform="translate(27.7 11)">
+          <use xlink:href="#led-body" />
+          ${led13 &&
+          svg`<circle cx="1.3" cy=".55" r="1.3" fill="#ffff80" filter="url(#ledFilter)" />`}
+        </g>
+
+        <!-- reset button -->
+        <g
+          id="reset-button"
+          stroke-width=".1"
+          paint-order="fill stroke"
+          @mousedown=${() => this.down()}
+          @mouseup=${() => this.up()}
+          @mouseleave=${() => this.up()}
+          @touchstart=${() => this.down()}
+          @touchend=${() => this.up()}
+        >
+          <rect x="24.3" y="6.3" width="1" height="4.8" filter="url(#solderPlate)" fill="#333" />
+          <rect x="23.54" y="6.8" width="2.54" height="3.8" fill="#ccc" stroke="#888" />
+          <ellipse cx="24.8" cy="8.7" rx="1" ry="1" fill="#eeb" stroke="#777" />
+          ${resetButton &&
+          svg`<circle cx="24.8" cy="8.7" r="1.5" fill="#f66" filter="url(#ledFilter)" />`}
```

Comment:
It makes more sense to change the color of the ellipse above the a darken shade when the button is pressed (we could also use a gradient to make the graphics a bit more realistic, similar to how the pushbutton works). 

Right now it seems like the button is illuminated whenever you press it.

Reply:
I'm happy to make those changes, but perhaps I should first explain why I did it this way.  At default zoom, the Nano is very small and the button is *tiny*.  I figured just reversing a gradient wouldn't be visible.  I have an MCU (a Teensy I think) with a red LED inside a similar tiny button, so it's not an invention of mine, just a little artistic license. :)

Reply:
Yes, a gradient may be to subtle in default zoom. The LED still seems a bit weird for me, how about just a darker shade for the round cap? e.g. #333